src/Symfony/Bundle/PhpAmqpBundle/PhpAmqpBundle.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\PhpAmqpBundle;
+
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+/**
+ * @author Samuel Roze <samuel.roze@gmail.com>
+ */
+class PhpAmqpBundle extends Bundle
```

Comment:
Here we spell `AMQP` as `Amqp` ... which I think it's the official policy of Symfony code syntax. So, in all other classes, we should replace `AMQP` by `Amqp` -> e.g. `AMQPEnvelope` -> `AmqpEnvelope`

Reply:
Unfortunately the `\AMQP*` classes such as `AMQPEnvelope` are coming from PHP's AMQP library 🤷‍♂️

Reply:
You can use aliased imports ;)